oval/resolution_test.go

```diff
@@ -0,0 +1,43 @@
+package oval
+
+import (
+	"encoding/xml"
+	"github.com/google/go-cmp/cmp"
+	"os"
+	"testing"
```

Comment:
```suggestion
	"encoding/xml"
	"os"
	"testing"

	"github.com/google/go-cmp/cmp"
```

Reply:
Yeah, I think this PR is good after this nit.